lib/AST/ClangTypeConverter.cpp

```diff
@@ -410,7 +410,19 @@ clang::QualType ClangTypeConverter::visitProtocolType(ProtocolType *type) {
 // Metatypes can be converted to Class when they are metatypes for concrete
```

Comment:
In case it wasn't clear from the summary, most of the changes here are from #30630.